src/components/ContentSidebar/SidebarSection.js

```diff
@@ -64,25 +83,40 @@ class SidebarSection extends PureComponent<Props, State> {
      */
     render() {
         const { isOpen }: State = this.state;
-        const { children, className, title }: Props = this.props;
+        const { children, className, title, hasCustomBranding }: Props = this.props;
 
         const sectionClassName = classNames(
             'bcs-section',
             {
-                'bcs-section-open': isOpen
+                'bcs-section-open': isOpen,
+                'bcs-section-has-custom-branding': hasCustomBranding
             },
             className
         );
 
         return (
-            <div className={sectionClassName}>
+            <section className={sectionClassName}>
```

Comment:
Revert this (and corresponding css) back to what it was originally for now.
I have the latest section designs.